rollup.config.js

```diff
@@ -0,0 +1,42 @@
+import babel from "rollup-plugin-babel";
```

Comment:
any specific reason to use rollup instead of webpack to build

Reply:
We have used [create-react-library](https://www.npmjs.com/package/create-react-library) npm to build this, and it uses rollup internally. I plan to switch to webpack later sometimes.